src/Product.php

```diff
@@ -191,7 +198,10 @@ public static function getStockByStore(\WC_Product $product): array {
     foreach ($stocks as $name => $types) {
       $stocks[$name]['online'] = ($is_backorder_allowed || $warehouse_stock) ? 'high' : 'none';
     }
-
+    if (!isset(static::$allVariationsStock)) {
+      static::$allVariationsStock = [];
+    }
```

Comment:
Please initialize the class property as an empty array instead.

Reply:
Updated in the last commit.